docs/development-workflow.md

```diff
@@ -51,6 +51,18 @@ or limit it as before with:
 DEBUG=calypso:* make run
 ```
 
+## Clearing browser data
+
+Seeing some strange errors that only happen on _your_ browser? You may need to clear your browser's local storage and database. Calypso saves a lot here and if you've changed the way your data is kept (especially in redux state), it can break things in unepxected ways during the transition.
+
+At the browser console line:
+
+```js
+localStorage.clear(); indexedDB.deleteDatabase( 'calypso' );
+```
+
+And this should clear out your database. Just make sure that in your changes you're not creating problems for production users!
```

Comment:
Yeah @coderkevin it'd definitely be preferable to have this be opt-in. Unfortunately if a handler isn't defined, default behavior is to return current state which causes things to be persisted.

How about if we added a whitelist? We could maybe add a manual one at first, then create a script if we can automate it.